.github/ISSUE_TEMPLATE/PLIP.md

```diff
@@ -3,6 +3,7 @@ name: "\U0001F680 PLIP"
 about: Plone Improvement Proposal
 title: ''
```

Comment:
To align with https://github.com/plone/plone.classicui/blob/main/.github/ISSUE_TEMPLATE/PLIP.md?plain=1#L4.

```suggestion
title: 'PLIP: '
```